lib3h/src/p2p/p2p_hackmode.rs

```diff
@@ -0,0 +1,218 @@
+#![allow(non_snake_case)]
+
+use crate::{
+    dht::{dht_event::DhtEvent, dht_trait::Dht, rrdht::rrdht},
+    p2p::{p2p_event::P2pEvent, p2p_trait::P2p},
+    transport::{Transport, TransportEvent},
+    transport_wss::TransportWss,
+};
+use holochain_lib3h_protocol::{DidWork, Lib3hResult};
+use std::collections::VecDeque;
+
+pub struct P2pHackmode {
```

Comment:
Ack! I vote for not doing modes again. I think it should be possible to just code toward our ultimate end goal, stubbing out or hacking things where we need to, but with an eye toward coming back and implementing the correct version when we get to it.

Reply:
hahaha. No worries, I just carried the name over from n3h. Gonna rename to `P2p` since that trait is actually useless.